pandas/core/base.py

```diff
@@ -929,11 +929,17 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
         """
         Return an ndarray of the maximum argument indexer.
 
+        If multiple values equal the maximum, the first row label with that
```

Comment:
If the maximum is achieved in multiple locations, the first such location is returned.

Reply:
Do you think we should go with location instead of row position?

Reply:
"row position" sounds good.  Definitely not "row label".